packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -16,6 +16,7 @@ import {ApiClientConfigParams, Prettify, RemoveStringIndex} from '../hooks/types
 import {BaseStorage, LocalStorage, CookieStorage, MemoryStorage, StorageType} from './storage'
 import {CustomerType} from '../hooks/useCustomerType'
 import {getParentOrigin, isOriginTrusted, onClient} from '../utils'
+import {SLAS_PRIVATE_SECRET_PLACEHOLDER} from '../constant'
```

Comment:
I wonder if commerce-sdk-react should be aware that the private client is a placeholder?

What if it just served as a pass-through, like commerce-sdk-isomorphic, and the knowledge of whether the private client is a placeholder or not is known elsewhere such as when the app sets the provider values?

Reply:
The above would mean we would need to pass in the client secret (or it's placeholder) via the provider however

Reply:
Marking this as resolved as discussions have lead to us applying this feedback. commerce-sdk-react will be agnostic to whether the client secret is a placeholder or not.